packages/block-editor/src/components/block-preview/README.md

```diff
@@ -0,0 +1,55 @@
+BlockPreview
+============
+
+BlockPreview allows you to preview any arbitrary HTML and CSS content, with a perfectly scaled thumbnail.
+
+## Usage
+
+Render the component passing in the required props:
+
+```jsx
+<BlockPreviewContent
+	blocks={ blocks }
+	srcWidth={ 400 }
+	srcHeight={ 300 }
+
+/>
+```
+
+## A note on source dimensions
+
+Please note that `srcWidth` and `srcHeight` refer to the _unscaled dimensions_ of what you mean to preview. 
+
+Think of the preview as a big source image, say 800x600 that's scaled down. So if the HTML you mean to preview looks good at 800x600, those are your source dimensionss. 
+
+A calculated `transform: scale( ...  )` value will be assigned to the thumbnail, so that it fits your _destination_ dimensions, which you set in CSS.
+
+
+## Props
+
+### `blocks`
+* **Type:** `array|object`
+* **Default:** `undefined`
+
+A block instance (object) or a blocks array you would like to render a preview.
+
+### `srcWidth`
+* **Type:** `Integer`
+* **Default:** `400`
+
+The unscaled dimension of the Block you are looking to preview. For example, if the Block looks good at `700x450` then you should set this value to `750`. See also `A note on source dimensions` above.
+
+### `srcHeight`
+* **Type:** `Integer`
+* **Default:** `300`
+
+The unscaled dimension of the Block you are looking to preview. For example, if the Block looks good at `700x450` then you should set this value to `450`. See also `A note on source dimensions` above.
+
+### `scaleFactor`
+* **Type**: `number`
+* **Default**: `0.9`
+
+### `scaleToFit`
+* **Type**: `bool`
+* **Default**: `false`
```

Comment:
This needs a refresh

Reply:
Agreed. I reckon we do a pass over all the docs once we know the API is solid.

Reply:
I've added a `scope` param to the DOM util and passed in a suitable scope to limit the `querySelector`.